Assets/Plugins/Source/Editor/Utility/Steam/Steamworks_Utility.cs

```diff
@@ -40,6 +41,48 @@ public static string GetSteamworksVersion()
             return "Steamworks not imported or not supported on platform";
 #else
             return Steamworks.Version.SteamworksSDKVersion;
+#endif
+        }
+
+        /// <summary>
+        /// This is to populate the SteamConfig's steamApiInterfaceVersionsArray. It needs to be configured exactly like this for Steamworks v1.58a onwards.
+        /// This value is identical to steam_api.h's pszInternalCheckInterfaceVersions value.
+        /// https://dev.epicgames.com/docs/en-US/api-ref/structs/eos-integrated-platform-steam-options
+        /// </summary>
+        /// <returns></returns>
+        public static List<string> GetSteamInterfaceVersions()
+        {
+#if DISABLESTEAMWORKS
+            return string.Empty;
```

Comment:
Return type mismatch if `DISABLESTEAMWORKS`
null might work here, just need to check if it affects the steam config window.

Reply:
Oh, yup. This used to return a string and I didn't come back to change this after it became a list. Good catch!